appveyor.yml

```diff
@@ -38,26 +33,14 @@ before_build:
 - cmd: SET PATH=%CONDA_PREFIX%;%CONDA_PREFIX%\Scripts;%PATH%
 - cmd: SET _prefix=%CONDA_PREFIX:~0,14%
 - cmd: SET _arch=%CONDA_PREFIX:~15,18%
+- cmd: conda install -y -c conda-forge libprotobuf=3.9.2 numpy
 - >
-  IF "%_arch%"=="x64"
-  (
-  conda install -y -c conda-forge libprotobuf=3.5.2=he51fdeb_1 numpy
-  )
-  ELSE
-  (
-  conda install -y -c conda-forge libprotobuf=3.5.2=vc14_0 numpy
-  )
 
   IF "%_prefix%"=="C:\Miniconda37"
   (
   set "PATH=%PATH%;%CONDA_PREFIX%\Library\bin"
   )
 
-  IF "%_prefix%"=="C:\Miniconda35"
-  (
-  pip install --quiet "ipython<7.10"
-  )
-
 - cmd: pip install --quiet pytest nbval numpy
```

Comment:
numpy can be removed from this command? It might be double installed.

Reply:
Removed at the protobuf installation line. I feel protobuf itself is difficult enough to occupy one line.